pkg/controller/clone-controller.go

```diff
@@ -311,17 +111,17 @@ func (c *CloneController) processPvcItem(pvc *v1.PersistentVolumeClaim) error {
 
 	// expectations prevent us from creating multiple pods. An expectation forces
 	// us to observe a pod's creation in the cache.
-	needsSync := c.podExpectations.SatisfiedExpectations(pvcKey)
+	needsSync := cc.podExpectations.SatisfiedExpectations(pvcKey)
 
 	// make sure not to reprocess a PVC that has already completed successfully,
 	// even if the pod no longer exists
-	phase, exists := pvc.ObjectMeta.Annotations[AnnClonePodPhase]
+	phase, exists := pvc.ObjectMeta.Annotations[AnnPodPhase]
```

Comment:
Is this correct?

Reply:
yes. The importer has the same annotation which can be common for both now.